src/test/ui/hello2021.rs

```diff
@@ -0,0 +1,6 @@
+// run-pass
+// edition:2021
+
+fn main() {
+    println!("hello, 2021");
+}
```

Comment:
It seems odd that this test passes - I would expect it to fail because -Zunstable-options isn't passed; --edition=2021 shouldn't be permitted without some additional opt-in to instability I think for now.

Reply:
Though I guess maybe this is just what we did before, though I find that a bit surprising. In particular we don't want to migrate anything in the ecosystem even if it's nightly only to using --edition=2021 yet, because that would be incompatible with the breaking change migrations we add in later on...

Reply:
Yeah it's just gated on nightly, not on `-Zunstable-options`. That's just what the behaviour of `Edition::is_stable() == false` already was. I'll update it to check for `-Zunstable-options` too.

Reply:
Done.